content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,172 @@
+in general, changes should not be made to rights provided to `system:` accounts some options 
+to harden cluster rights exist :- 
+
+- Review bindings for the `system:unauthenticated` group and remove where possible, as this gives 
+  access to anyone who can contact the API server at a network level.
+- Avoid the default auto-mounting of service account tokens by setting
+  `automountServiceAccountToken: false`. For more details, see
+  [using default service account token]/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server).
+  Setting this value for a Pod will overwrite the service account setting, workloads
+  which require service account tokens can still mount them.
+
+### Periodic review
+
+It is vital to periodically review the Kubernetes RBAC settings for redundant entries and 
+possible privilege escalations.
+If an attacker is able to create a user account with the same name as a deleted user,
+they can automatically inherit all the rights of the deleted user, especially the
+rights assigned to that user.
+
+## Kubernetes RBAC - privilege escalation risks 
+
+Within Kubernetes RBAC there are a number of privileges which, if granted, can allow a user or a service account
+to escalate their privileges in the cluster or affect systems outside the cluster.
+
+This section is intended to provide visibility of the areas where cluster operators 
+should take care, to ensure that they do not inadvertantly allow for more access to clusters than intended.
+
+### Listing secrets
+
+It is generally clear that allowing **Get** access on Secrets will allow a user to read their contents.
+It is also important to note that the **List** access also effectively allows for users to reveal the Secret contents.
+When a List response is returned (for example, via `kubectl get secrets -A -o yaml`), the response
+includes the contents of all Secrets.
+
+### Workload creation
+
+Users who are able to create workloads (either Pods, or
+[workload resources](/docs/concepts/workloads/controllers/) that manage Pods) will
+be able to gain access to the underlying node unless restrictions based on the Kubernetes
+[Pod Security Standards](/docs/concepts/security/pod-security-standards/) are in place.
+
+Users who can run privileged Pods can use that access to gain node access and potentially to
+further elevate their privileges. Where you do not fully trust a user or other principal
+with the ability to create suitably secure and isolated Pods, you should enforce either the
+**Baseline** or **Restricted** Pod Security Standard.
+You can use [Pod Security admission](/docs/concepts/security/pod-security-admission/)
+or other (third party) mechanisms to implement that enforcement.
+
+You can also use the deprecated [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) mechanism
+to restrict users' abilities to create privileged Pods.
```

Comment:
I'd avoid mentioning this in new documentation, or if we mention it, explicitly call out the 1.25 removal

Reply:
@sftim What's your preference here, should we just remove this bit, or I can add an explicit mention of the removal in 1.25. On the one hand removing it makes sense as it is deprecated. On the other hand I know how many really old clusters are still running for which PSP is likely a good way to address this kind of issue...

Reply:
I would add it here and then clean the new doc up for the _dev-1.25_ branch.

Reply:
righto. Added a specific note of version deprecation here and we can clean in 1.25 docs :)